meeting-notes/2018/2018-10-01--go-core-dev-team-weekly.md

```diff
@@ -0,0 +1,163 @@
+# 💫Golang Core Dev Team Weekly Sync 🙌🏽 October 1, 2018
+
+- **Lead:** Erik Ingenito
+- **Notetaker:**
+  - momack2ck2
+- **Attendees:**
+  - _attendee names..._
+  - momack2
+  - bigs
+ - michaelavila
+  - kevina
+  - Steven
+  - Mikael
+  - djdv
+- **Recording:** _add link to recording once it's online_
+
+## Agenda
+
+- Ask everyone to put their name into the list of attendees
+- @momack2 - meta agenda plan - how to use our time most effectively on this call
+- Round of updates
+  - What have you accomplished since the last Weekly?
+  - Were there any blockers? If so, which ones? Is it still blocked? Why?
+  - What is the next important thing you should focus on?
+- Ask for general questions. Could be things like:
+  - I'm stuck with something, I don't know who to ask. Who knows who to ask?
+  - Who can help me with xyz?
+- Plan this week
+  - Select issues to work on
+- Review remaining issues if there is time left
+
+
+## Notes
+ - Molly: Proposal to only call out blockers and top-level progress to save time for sync OKR planning
+ - Erik: working on grooming the waffle board
+ - Hannah: questions about the sharded directory tests - test data itself not setup for the new implementation. Need Steven/Jeromy to double check that the implementation is correct and the tests are wrong
+ - Kevin: need review on outstanding PRs, in particular the cid subcommand 
+ - 
```

Comment:
please add " - All: Synchronously walked through https://github.com/ipfs/go-ipfs/pull/5474 to rephrase OKRs, add owners, and set priorities"